tests/vue/options/option-item-block.test.js

```diff
@@ -0,0 +1,246 @@
+/* eslint-env jest */
+/* eslint-disable no-unused-vars */
+
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import AppController from '@/lib/controllers/app-controller.js'
+import OptionItemBlock from '@/vue/options/option-item-block.vue'
+
+import Vuex from "vuex"
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+let appC, localTextEditorOptions
+
+describe('option-item-block.test.js', () => {
+  console.error = function () {}
+  console.log = function () {}
+  console.warn = function () {}
+
+  beforeEach(async () => {
+    jest.spyOn(console, 'error')
+    jest.spyOn(console, 'log')
+    jest.spyOn(console, 'warn')
+
+    appC = new AppController({
+      appId:'alpheios-alignment-editor'
+    })
+    
+    appC.defineStore()
+    appC.defineL10Support()
+    appC.defineNotificationSupport(appC.store)
+
+    await appC.defineSettingsController()
+    await appC.settingsC.init()
+    await appC.settingsC.uploadRemoteSettings()

```

Comment:
I would prefer that we not have a dependency on the live tokenizer service for this test. The default settings of the tokenizer might change, and that would break these tests without any change in this code. 

can you instead write these tests with a local fixture for one of each type of setting?

Reply:
I have updated all tests (besides a direct test for uploadRemoteSettings method).
Fixed